app/views/curation_concern/generic_works/new.html.erb

```diff
@@ -1,11 +1,12 @@
 <% content_for :page_title, curation_concern_page_title(curation_concern) %>
 <% content_for :page_header do %>
 
-<h2>Submit Your <%= curation_concern.human_readable_type %></h2>
-<% end %>
+<div class="alert">
```

Comment:
Please change class to "help-alert." This is a new custom class that will live in the app.